src/test/java/hudson/plugins/mercurial/MercurialStatusTest.java

```diff
@@ -35,18 +35,18 @@
 public class MercurialStatusTest {
   
   @Test public void testLooselyMatches() throws URISyntaxException {
-      assertTrue( MercurialStatus.looselyMatches(new URI("ssh://somehost/"), "ssh://somehost/"));
-      assertTrue( MercurialStatus.looselyMatches(new URI("http://somehost/"), "http://somehost/"));
+      assertTrue( MercurialStatus.looselyMatches(new URI("ssh://somehost/"), "ssh://somehost"));
+      assertTrue( MercurialStatus.looselyMatches(new URI("http://somehost"), "http://somehost/"));
```

Comment:
Generally prefer that a bug fix _add_ tests without removing existing tests, but I think in this case it is OK since you seem to be strengthening the existing assertions.